docs/iris/src/userguide/index.rst

```diff
@@ -30,7 +30,6 @@ User guide table of contents
    saving_iris_cubes.rst
    navigating_a_cube.rst
    subsetting_a_cube.rst
-   real_and_lazy_data.rst
```

Comment:
This was originally written with the user in mind, rather than a more technical white paper.